src/scripts/operations/chooseOperation.js

```diff
@@ -0,0 +1,26 @@
+import compute from '../compute';
+import { operationOperand } from '../calculator';
+
+export default class ChooseOperation {
+  constructor(operation) {
+    this.operation = operation;
+  }
+
+  // eslint-disable-next-line consistent-return
+  execute(data) {
+    this.data = data;
+    if (this.data.currentNumber === '') {
+      return this.data.previousNumber === '' ? data : { ...this.data, operation: this.operation };
+    // eslint-disable-next-line no-else-return
+    } else if (!Object.values(data).includes('')) {
+      compute(this.data);
+      operationOperand.innerHTML = this.operation;
+    } else {
+      return { previousNumber: data.currentNumber, operation: this.operation };
+    }
```

Comment:
Не используй змеевидные if else if else if else if else в таких случаях код очень нагромождается и его неприятно читать. 

Лучше выделять отдельные блоки if и делать из них return 

```suggestion
    if (this.data.currentNumber === '') {
      return this.data.previousNumber === '' ? data : { ...this.data, operation: this.operation };
    } 
    
    if (!Object.values(data).includes('')) {
      compute(this.data);
      operationOperand.innerHTML = this.operation;
      return;
    } 
    
    return { previousNumber: data.currentNumber, operation: this.operation };
```

В итоге я могу читать код сверху вниз без большого нагромождения